HBFA/UefiHostFuzzTestCasePkg/TestCase/OvmfPkg/Library/TdxStartupLib/TestHobList.c

```diff
@@ -23,28 +23,233 @@
 #include <Library/PeilessStartupLib.h>
 #include <Library/HobLib.h>
 #include <IndustryStandard/Tdx.h>
+#include <Pi/PrePiHob.h>
 
 #define GET_GPAW_INIT_STATE(INFO)  ((UINT8) ((INFO) & 0x3f))
 
 #define TOTAL_SIZE   (512 * 1024)
 #define BLOCK_SIZE   (512)
 #define IO_ALIGN     (1)
 
+// TODO: copied from SecTdxHelper.c, temporary hack for static functions
```

Comment:
remove todo

Reply:
fixed in [0148b9e](https://github.com/tianocore/edk2-staging/pull/386/commits/0148b9e1fc1d0947610f4bb1da22a6b373d9258e)